src/main/java/de/cismet/cids/custom/wunda_blau/search/server/QsgebStatusLightweightSearch.java

```diff
@@ -0,0 +1,241 @@
+
+import de.cismet.connectioncontext.ConnectionContext;
+import de.cismet.connectioncontext.ConnectionContextStore;
+
+/**
+ * Builtin Legacy Search to delegate the operation getLightweightMetaObjectsByQuery to the cids Pure REST Search API.
+ *
+ * @author   sandra
+ * @version  $1.0$, $31.05.2018$
+ */
+@ServiceProvider(service = RestApiCidsServerSearch.class)
+public class QsgebStatusLightweightSearch extends AbstractCidsServerSearch implements RestApiCidsServerSearch,
+    LightweightMetaObjectsSearch,
+    ConnectionContextStore {
+
+    //~ Static fields/initializers ---------------------------------------------
+
+    private static final Logger LOG = Logger.getLogger(QsgebStatusLightweightSearch.class);
+
+    private static final String TABLE_QSGEB_STATUS = "qsgeb_status"; 
+    public static final String FIELD__STATUS_ID = "status.id";
+
+    //~ Enums ------------------------------------------------------------------
+
+    /**
+     * DOCUMENT ME!
+     *
+     * @version  $Revision$, $Date$
+     */
+    public enum SearchFor {
+
+        //~ Enum constants -----------------------------------------------------
+
+        STATUS
+    }
+
+    //~ Instance fields --------------------------------------------------------
+
+    private ConnectionContext connectionContext = ConnectionContext.createDummy();
+
+    @Getter private final SearchInfo searchInfo;
+    @Getter @Setter private SearchFor searchFor;
+    @Getter @Setter private Integer statusId;
+    @Getter @Setter private String representationPattern;
+    @Getter @Setter private String[] representationFields;
+
+    //~ Constructors -----------------------------------------------------------
+
+    /**
+     * Creates a new LightweightMetaObjectsByQuerySearch object.
+     */
+    public QsgebStatusLightweightSearch() {
+        this.searchInfo = new SearchInfo(
+                this.getClass().getName(),
+                this.getClass().getSimpleName(),
+                "Builtin Legacy Search to delegate the operation getLightweightMetaObjectsByQuery to the cids Pure REST Search API.",
+                Arrays.asList(
+                    new SearchParameterInfo[] {
+                        new MySearchParameterInfo(FIELD__STATUS_ID, Type.INTEGER),
+                        new MySearchParameterInfo("searchFor", Type.UNDEFINED),
+                        new MySearchParameterInfo("representationPattern", Type.STRING, true),
+                        new MySearchParameterInfo("representationFields", Type.STRING, true)
+                    }),
+                new MySearchParameterInfo("return", Type.ENTITY_REFERENCE, true));
+    }
+
+    /**
+     * Creates a new StatusArtLightweightSearch object.
+     *
+     * @param  searchFor              DOCUMENT ME!
+     * @param  representationPattern  DOCUMENT ME!
+     * @param  representationFields   DOCUMENT ME!
+     */
+    public QsgebStatusLightweightSearch(final SearchFor searchFor,
```

Comment:
`searchFor` is unnecessary

on the other side, statusId is missing in the constructor but would make sense to be there.